### PR TITLE
ECR-2752 [FE] Update cryptocurrency-java demo frontend to EJB 0.4

### PR DESCRIPTION
**Pull request by KateNegrienko**
## Overview
Update cryptocurrency-java demo frontend to EJB 0.4

---
See: https://jira.bf.local/browse/ECR-2752

### Definition of Done

- [ ] There are no TODOs left in the code
- [ ] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [ ] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [ ] Public API has Javadoc
- [ ] Method preconditions are checked and documented in the Javadoc of the method
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding-cryptocurrency-demo/frontend/src/proto/helpers.proto [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package exonum;
+
+message Hash { bytes data = 1; }
```
Comment (MakarovS) [inline exonum-java-binding-cryptocurrency-demo/frontend/src/proto/helpers.proto]: Is there an easy way to reuse the same proto package in both core/binding/demo? As we have same messages defined in many places.

Comment (KateNegrienko) [reply exonum-java-binding-cryptocurrency-demo/frontend/src/proto/helpers.proto]: Sorry, my mistake. I fixed.

**Comment by dmitry-timofeev** [comment]
I'd keep this PR open until we resolve #574 

Alex Bogatov has verified that the backend converts transactions submitted this way correctly :tada:

File: exonum-java-binding-cryptocurrency-demo/frontend/src/plugins/blockchain.js [hunk 2]
```diff
@@ -1,49 +1,15 @@
 import * as Exonum from 'exonum-client'
 import axios from 'axios'
-import * as Protobuf from 'protobufjs/light'
 import bigInt from 'big-integer'
+import * as proto from '../proto/proto.js'
```
Comment (bullet-tooth) [inline exonum-java-binding-cryptocurrency-demo/frontend/src/plugins/blockchain.js]: there is no such file

**Comment by dmitry-timofeev** [comment]
Pushed as 96368b025286061e0ab2daf855cf4cedd8bc0030 to [`new-message-format-1`](https://github.com/exonum/exonum-java-binding/commits/new-message-format-1)

**Pull request closed without merging**